src/main/java/com/softserveinc/ita/pageobjects/util/RandomUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.softserveinc.ita.pageobjects.util;
+
+import lombok.experimental.UtilityClass;
+
+import java.util.Random;
+
+import static java.lang.Math.pow;
+
+@UtilityClass
+public class RandomUtil {
+    public int getRandomNumber(int length) {
```

Comment:
make it static

Reply:
All members of a utility class are automatically marked as static with lombok @UtilityClass annotation